mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -134,6 +138,7 @@ pub enum SwapMsg {
     TakerFee(Vec<u8>),
     MakerPayment(Vec<u8>),
     TakerPayment(Vec<u8>),
+    WatcherMessage(Box<WatcherData>),
```

Comment:
Please create a separate `SwapWatcherMsg` struct. This way, we will avoid changing `lp_swap::process_msg`. I also think we should keep `Pubsub topic:single message` 1:1 relation in all cases for better code composition.

Reply:
Do you mean I should keep the `swap_topic` and `watcher_topic` functions seperate as it currently is?

Reply:
Yes, please. I also left a comment here https://github.com/KomodoPlatform/atomicDEX-API/pull/1457#discussion_r962619082